internal/pkg/describe/service.go

```diff
@@ -280,6 +286,22 @@ func (d *appRunnerServiceDescriber) ServiceARN() (string, error) {
 	return "", fmt.Errorf("no App Runner Service in service stack")
 }
 
+func (d *appRunnerServiceDescriber) vpcIngressConnectionARN() (*string, error) {
```

Comment:
nit: why do we return pointer? does `""` mean different thing than `nil` here?

Reply:
I think it's possible that it's nil if there is no `VpcIngressConnection` resource, but to address Wanxian's feedback. I wonder if this method signature is a bit more explicit?
```suggestion
// vpcIngressConnectionARN returns an errResourceNotFound error if the vpc ingress does not exist.
func (d *appRunnerServiceDescriber) vpcIngressConnectionARN() (string, error) {
```
We can return a non-pointer, but instead return a custom error `errResourceNotFound` if it doesn't exist.
Then the caller has to check if this is the error returned and skip or whatever